JustTrack/Classes/JustTrack/Tracking.swift

```diff
@@ -212,30 +211,27 @@ public class EventTracking: NSObject {
     fileprivate func restoreUncompletedTracking() -> Int {
         var operations: NSMutableDictionary
         guard let outData = UserDefaults.standard.data(forKey: EventTracking.kPersistentStorageName),
-              let dataDictionary = NSKeyedUnarchiver.unarchiveObject(with: outData) as? [AnyHashable: Any]
-        else {
+              let dataDictionary = NSKeyedUnarchiver.unarchiveObject(with: outData) as? [AnyHashable: Any] else {
```

Comment:
I think this function is now deprecated https://developer.apple.com/documentation/foundation/nskeyedunarchiver/1413894-unarchiveobject

Reply:
yes, it will be replaced/removed later. as you may see there are multiple open PRs with some changes. I have not replaced yet `NSKeyedUnarchiver.unarchiveObject` but this is in my todo list